src/carvel/config/values/values.yml

```diff
@@ -112,5 +112,5 @@ scdf:
         enabled: false
         image:
           repository: micrometermetrics/prometheus-rsocket-proxy
-          tag: 2.0.0-M2
```

Comment:
It would be nice to find a way to templatize this variable so that we set it in one place (even in a few places would be better than in every yml file in the installers).